homeassistant/components/elevenlabstts/manifest.json

```
@@ -0,0 +1,11 @@
+{
+  "domain": "elevenlabstts",
+  "name": "ElevenLabs Text-to-Speech (TTS)",
+  "codeowners": ["@sorgfresser"],
+  "config_flow": true,
+  "documentation": "https://www.home-assistant.io/integrations/elevenlabs",
+  "integration_type": "service",
+  "iot_class": "cloud_push",
```

It's cloud poll, as we have to request data from the cloud.
```suggestion
  "iot_class": "cloud_poll",
```